pgclone/run.py

```diff
@@ -8,11 +8,24 @@
 from pgclone import exceptions, logging
 
 
-def shell(cmd, ignore_errors=False, env=None):
+def _is_pipefail_supported():
```

Comment:
`set -o pipefail` is supported in most POSSIX shell implementations at this point, it was added to the spec in 2022 - but the user could still be missing it